src/configure_transform.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+
+import os
+import re
+import sys
+
+define_re = re.compile("#define ([A-Z_]+) (.*)")
+
```

Comment:
If it's kosher to have a random python script, then I don't see why not! Two more things to think about. Would it be more appropriate to have some kind of organizational logic other than just having in `src`? If we do decide to move forward with python for this, would you mind some help to make it a bit more function based and script-ish? (Eg, using argparse instead of sys.argv, and having functions. *raises hand* I would be glad to help with this!! :OD